x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/index.tsx

```diff
@@ -107,9 +144,10 @@ export function MobileStats({
         defaultMessage: 'HTTP requests',
       }),
       icon: getIcon('kubernetesPod'),
-      value: data?.currentPeriod?.requests?.value ?? NaN,
+      value: data?.currentPeriod.requests?.value ?? NaN,
+      extra: getComparisonValueFormatter(data?.previousPeriod.requests?.value),
       valueFormatter: (value: number) => valueFormatter(value),
-      trend: data?.currentPeriod?.requests?.timeseries ?? [],
+      trend: data?.currentPeriod.requests.timeseries,
```

Comment:
could this still be undefined? From the line 147 I'm getting that `data.currentPeriod.request?` could be undefined. Is the component prepared for this?